Google.Authenticator/TwoFactorAuthenticator.cs

```diff
@@ -17,209 +16,222 @@ namespace Google.Authenticator
+            string accountTitleNoSpaces, 
+            string accountSecretKey,
+            bool secretIsBase32, 
+            int qrPixelsPerModule = 3)
         {
-            byte[] key = secretIsBase32 ? Base32Encoding.ToBytes(accountSecretKey) : Encoding.UTF8.GetBytes(accountSecretKey);
-            return GenerateSetupCode(issuer, accountTitleNoSpaces, key, QRPixelsPerModule);
+            var key = secretIsBase32
+                ? Base32Encoding.ToBytes(accountSecretKey)
+                : Encoding.UTF8.GetBytes(accountSecretKey);
+            
+            return GenerateSetupCode(issuer, accountTitleNoSpaces, key, qrPixelsPerModule);
         }
 
         /// <summary>
         /// Generate a setup code for a Google Authenticator user to scan
         /// </summary>
-        /// <param name="issuer">Issuer ID (the name of the system, i.e. 'MyApp'), can be omitted but not recommended https://github.com/google/google-authenticator/wiki/Key-Uri-Format </param>
+        /// <param name="issuer">Issuer ID (the name of the system, i.e. 'MyApp'), can be omitted but not
+        /// recommended https://github.com/google/google-authenticator/wiki/Key-Uri-Format </param>
         /// <param name="accountTitleNoSpaces">Account Title (no spaces)</param>
         /// <param name="accountSecretKey">Account Secret Key as byte[]</param>
-        /// <param name="QRPixelsPerModule">Number of pixels per QR Module (2 = ~120x120px QRCode, should be 10 or less)</param>
+        /// <param name="qrPixelsPerModule">Number of pixels per QR Module
+        /// (2 = ~120x120px QRCode, should be 10 or less)</param>
+        /// <param name="generateQrCode"></param>
         /// <returns>SetupCode object</returns>
-        public SetupCode GenerateSetupCode(string issuer, string accountTitleNoSpaces, byte[] accountSecretKey, int QRPixelsPerModule = 3, bool generateQrCode = true)
-        {
-            if (String.IsNullOrWhiteSpace(accountTitleNoSpaces)) { throw new NullReferenceException("Account Title is null"); }
+        public SetupCode GenerateSetupCode(
+            string issuer,
+            string accountTitleNoSpaces,
+            byte[] accountSecretKey,
+            int qrPixelsPerModule = 3,
+            bool generateQrCode = true)
+        {
+            if (string.IsNullOrWhiteSpace(accountTitleNoSpaces))
+                throw new NullReferenceException("Account Title is null");
+            
             accountTitleNoSpaces = RemoveWhitespace(Uri.EscapeUriString(accountTitleNoSpaces));
-            string encodedSecretKey = Base32Encoding.ToString(accountSecretKey);
-            string provisionUrl;
-            if (String.IsNullOrWhiteSpace(issuer))
-            {
-                provisionUrl = String.Format("otpauth://totp/{0}?secret={1}", accountTitleNoSpaces, encodedSecretKey.Trim('='));
-            }
-            else
-            {
-                //  https://github.com/google/google-authenticator/wiki/Conflicting-Accounts

```

Comment:
Please don't remove these comments

Reply:
Sorry, that was not on purpose. Returned it.